### PR TITLE
Update project definition

### PR DESCRIPTION
**Pull request by aboalnaga**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

The phrase __A project, as generated by the ng new__ is confusing and could be misleading.

Issue Number: N/A

## What is the new behavior?
Elaborate briefly on the default behavior of `ng new` command

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by aboalnaga** [comment]
I signed it!

File: aio/content/guide/glossary.md [hunk 1]
```diff
@@ -647,7 +647,7 @@ See [Browser Support](guide/browser-support) for polyfills that support particul
 
 In the Angular CLI, a standalone application or [library](#library) that can be created or modified by a CLI command.
 
-A project, as generated by the [`ng new`](cli/new), contains the set of source files, resources, and configuration files that you need to develop and test the application using the CLI. Projects can also be created with the `ng generate application` and `ng generate library` commands.
+A project contains set of source files, resources, and configuration files that you need to develop and test the application using the CLI. Projects can be created with the `ng generate application`, `ng generate library` commands, and also [`ng new`](cli/new) command, which by default creates an initial application project with the same name of the [workspace](#workspace).
```
Comment (jbogarthyde) [inline aio/content/guide/glossary.md]: In light of new terminology used in the workspace configuration page, how about something like this:
`By default, the [`ng new`](cli/new) command creates an initial application project  at the root of the [workspace](#workspace). You can add projects to an existing workspace with the `ng generate application`, `ng generate library` commands.  See [Multiple Projects](guide/file-structure#multiple-projects).`

**Review by jbogarthyde — CHANGES_REQUESTED** [review]
We can point to additional configuration doc that has been added since the last Glossary edit.

**Comment by jbogarthyde** [comment]
You need to switch the *PR title* to the *commit message*.  It's the commit message that needs to use the "docs: xx" format. Your current commit message, "Update project definition", makes a better PR title :-).

File: aio/content/guide/glossary.md (hunk 1 above)
Comment (aboalnaga) [reply aio/content/guide/glossary.md]: I think, in light of the first statement -in which the omission of _project_ is more confusing than elegant-, specifying the methods of direct creation is more pressing, than a side-effect of using `ng new` command.

I'm all for adding a link to _multiple project_, also if there there's a link that mentions the relation between a _workspace_, a _project_ and a _module_, I think will make things more clearer for a newbie like myself.

Sorry, I failed to understand what you meant by _new terminology in the workspace configuration page_.

**Comment by aboalnaga** [comment]
I'm confused, should I close this PR, create a new commit with the updates, then create a new PR, or are there a way to add a commit to this PR.

**Comment by aboalnaga** [comment]
> I'm confused, should I close this PR, create a new commit with the updates, then create a new PR, or are there a way to add a commit to this PR.

Sorry just noticed "_Add more commits by pushing to the ..._"

File: aio/content/guide/glossary.md (hunk 1 above)
Comment (jbogarthyde) [reply aio/content/guide/glossary.md]: The current best practice recommendation is for most developers and new users to use ‘ng new’ to create a workspace with a single root-level app for each app that you want to work with. For an organization that is using a monorepo, an admin could create a workspace with no root-level project, and developers could add their own projects with ‘ng generate’. Library developers would also create a multi-project workspace— but these are considered exceptional cases, not the basic or standard model we want to teach.

**Comment by jbogarthyde** [comment]
You can change the commit message for the existing commit— look it up 🙂

**Comment by jbogarthyde** [comment]
> I signed it!

Apparently not, CLA is still failing -- are you sure you used the same email that you use for your git account?

**Comment by aboalnaga** [comment]
> > I signed it!
> 
> Apparently not, CLA is still failing -- are you sure you used the same email that you use for your git account?

Thank you :)

**Comment by aboalnaga** [comment]
Should I, close this PR?

**Comment by kapunahelewong** [comment]
Hi @aboalnaga, I just saw this. If you'd like to open another PR at any point (or re-open this one), I'd be happy to help you through the process. Feel free to tag me.

**Pull request closed without merging**